UnityProject/Assets/Scripts/Systems/Explosions/ExplosionManager.cs

```diff
@@ -60,13 +60,17 @@ private void OnEnable()
 		{
 			if(Application.isEditor == false && NetworkServer.active == false) return;
 
+			if (CustomNetworkManager.IsServer == false) return;
```

Comment:
Have you tested to make sure this doesn't break anything?

Reply:
also bod

Reply:
This should be fine since all explosion stuff should be done server side